### PR TITLE
Apply ServiceTreeEntryID via local.env

### PR DESCRIPTION
**Pull request by georearl**
this PR allows a user to set a value (ServiceTreeEntryID) in local.env which then applies that value to Service management reference  property for the created application registrations

### CODE REVIEW

File: scripts/environments/local.env.example [hunk 1]
```diff
@@ -145,3 +145,6 @@ export ENABLE_DEV_CODE=false
 
 # If you are deploying the solution with the ability to use the Tabular Data Assistant feature, you can set the following values to configure max file size of csv files to be uploaded.
 export MAX_CSV_FILE_SIZE="20" #default is 20MB
+
+# Set a value here if you are required to apply a valye in Entra for Service management references
```
Comment (dayland) [inline scripts/environments/local.env.example:149]: typo

File: infra/outputs.tf [hunk 2]
```diff
@@ -188,4 +188,7 @@ output "AZURE_ARM_MANAGEMENT_API" {
 }
 output "MAX_CSV_FILE_SIZE" {
   value = var.maxCsvFileSize
+}
```
Comment (dayland) [inline infra/outputs.tf:191]: why do we need to output this? It isn't used downstream...

File: scripts/json-to-env.sh [hunk 3]
```diff
@@ -118,6 +118,10 @@ jq -r  '
         {
             "path": "MAX_CSV_FILE_SIZE",
             "env_var": "MAX_CSV_FILE_SIZE"
+        },
+        {
+            "path": "SERVICE_MANAGEMENT_REFERENCE",
```
Comment (dayland) [inline scripts/json-to-env.sh:123]: This doesn't need to be mapped at output as it isn't used

File: docs/deployment/deployment.md [hunk 4]
```diff
@@ -80,6 +80,9 @@ DEFAULT_LANGUAGE | Yes | Use the parameter to specify the matching ENV file loca
...
 ENABLE_DEV_CODE | No | Defaults to `false`. It is not recommended to enable this flag, it is for development testing scenarios only.
 APPLICATION_TITLE | No | Defaults to "". Providing a value for this parameter will replace the Information Assistant's title in the black banner at the top of the UX.
+MAX_CSV_FILE_SIZE | No | Defaults to 20 (MB's) for the maximum file size for an uploaded CSV
+SERVICE_MANAGEMENT_REFERENCE | No| Defaults to "". This value is applied to Service management reference entries if required by your organization.
```
Comment (dayland) [inline docs/deployment/deployment.md:84]: Update comment to be more clear.. 
"Sets the service management reference value on Azure Entra objects created by Information Assistant if required by your organization"

File: scripts/json-to-env.sh (hunk 3 above)
Comment (georearl) [reply scripts/json-to-env.sh:123]: fixed

File: docs/deployment/deployment.md (hunk 4 above)
Comment (georearl) [reply docs/deployment/deployment.md]: fixed

**Review by dayland — APPROVED** [review]

**Pull request merged**